### ISSUE TITLE
Länsförsäkringar Funds and Pension

### ISSUE THREAD

**Issue by Johankp** [liato/android-bankdroid#274]
Skulle gärna vilja kunna se mina fonder samt IPS på Länsförsäkringar i appen :)

**Comment by goober** [comment]
@Johankp Is this request still valid?

**Comment by Johankp** [comment]
Yes It is. Can tell you that Saabkortet isn't working because it is controlled by bankid.

**Comment by goober** [comment]
Saab was removed by #419 and will be included in the upcoming release.

**Comment by janstad** [comment]
Please also add support for ISK. It's accessible in the app through standard pin login.

**Comment by fredrike** [comment]
@liato How did you find the `API_BASEURL + "account/bytype"` urls?
